pvr.hts/resources/language/resource.language.en_gb/strings.po

```diff
@@ -50,7 +50,25 @@ msgctxt "#30007"
 msgid "Response timeout in seconds"
 msgstr ""
 
-#empty strings from id 30008 to 30099
+msgctxt "#30008"
+msgid "Advanced Settings"
+msgstr ""
+
+#empty strings from id 30009 to 30049
+
+msgctxt "#30050"
+msgid "Auto Recordings"
+msgstr ""
+
+msgctxt "#30051"
+msgid "Use timer start time as approximate"
```

Comment:
I'd use tvheadend webui terminology here. Thus, it should be more understandable to users familiar with tvheadend webui.